src/main/kotlin/com.bugsnag.android.gradle/BugsnagPlugin.kt

```diff
@@ -160,7 +160,7 @@ class BugsnagPlugin : Plugin<Project> {
             it.projectDir = project.projectDir
             it.rootDir = project.rootDir
             it.sharedObjectPath = bugsnag.sharedObjectPath
-            addTaskToExecutionGraph(it, variant, output, project, bugsnag, bugsnag.isAutoUpload)
+            addTaskToExecutionGraph(it, variant, output, project, bugsnag, true)
```

Comment:
Is this supposed to always be true, rather than `bugsnag.isUploadJvmMappings`?

Reply:
This is already gated by `bugsnag.isUploadNdkMappings` in the `registerBugsnagTasksForVariant` method